templates/boot-fastboot/generic-fastboot-boot-template.jinja2

```diff
@@ -0,0 +1,85 @@
+{%- extends 'base/kernel-ci-base.jinja2' %}
+{%- block protocols %}
+protocols:
+  lava-lxc:
+    name: lxc-{{ platform }}-art-conv
+    template: debian
+    distribution: debian
+    release: jessie
+    mirror: http://mirror.bytemark.co.uk/debian
+{%- endblock %}
+
+{% block actions %}
+actions:
+{%- block deploy_lxc %}
+
+  - deploy:
+      namespace: tlxc
+      timeout:
+        minutes: 5
+      to: lxc
+      packages:
+    - android-tools-adb
+    - android-tools-fastboot
+    - mkbootimg
+    - skales
+{%- endblock %}
+{%- block boot_lxc %}
+
+  - boot:
+      namespace: tlxc
+      prompts:
+      - 'root@(.*):/#'
+      timeout:
+        minutes: 5
+      method: lxc
+{%- endblock %}
+{%- block deploy_url %}
+
+  - deploy:
+      timeout:
+        minutes: 40
+      namespace: droid
+      to: download
+      images:
+      kernel:
+        url: {{ kernel_url }}
+{%- block kernel_image_type %}
+        type: {{ kernel_image.lower() }}
+{%- endblock %}
+{%- if initrd_url %}
+      ramdisk:
+        url: {{ initrd_url }}
+        compression: gz
+{%- endif %}
+{%- if dtb_url %}
+      dtb:
+        url: {{ dtb_url }}
+{%- endif %}
+{%- endblock %}
+{%- block test_lxc %}
+
+  {% include "/boot-fastboot/" + device_type + ".jinja2" %}
+{%- endblock %}
+{%- block deploy_boot %}
+
+  - deploy:
+      timeout:
+        minutes: 40
+      to: fastboot
+      namespace: droid
+      images:
+        boot:
+          url: lxc:///boot.img
```

Comment:
This `boot.img` filename  is dependent on the device-type specific template in `/boot-fastboot` so a variable should be used here.